api/internal/generators/secret_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	. "sigs.k8s.io/kustomize/api/internal/generators"
 	"sigs.k8s.io/kustomize/api/kv"
 	"sigs.k8s.io/kustomize/api/loader"
-	valtest_test "sigs.k8s.io/kustomize/api/testutils/valtest"
+	valtest "sigs.k8s.io/kustomize/api/testutils/valtest"
```

Comment:
So if you go to the package where these test helpers are defined, the name of the package is actually valtest_test. I think this is pretty standard for test packages (so that it gets compiled separately from the main package). 

I'm not really sure it matters whether we call import `valtest` or `valtest_test` - kind of seems like we are trying to decide between using the package name or the directory name. To me using the actual package name `valtest_test` seems to make more sense.

I think what _does_ matter is that we choose one pattern and stick to it, so I'd be inclined to leave it as `valtest_test` since that seems to have been the convention so far. Again I really don't think it matters too much so I could be convinced otherwise if someone feels there is a strong reason to change it.

Reply:
This makes sense to me. I'll close this for the time being, as I mainly opened so we could discuss.

/close